src/test/java/decodex/modules/binary/BinaryDecoderTest.java

```diff
@@ -0,0 +1,54 @@
+package decodex.modules.binary;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import decodex.data.Data;
+import decodex.data.exception.ModuleException;
+import org.junit.jupiter.api.Test;
+
+public class BinaryDecoderTest {
+
+    BinaryDecoder binaryDecoder = new BinaryDecoder();

```

Comment:
Perhaps move this into the individual tests instead? Following the other module tests.